<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog3</class>
 <widget class="QDialog" name="Dialog3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(134,237,231);</string>
  </property>
  <widget class="QPushButton" name="pushButton1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:focus { background-color: rgb(244,236,2); }</string>
   </property>
   <property name="text">
    <string>1、发送分站保存的位移传感器数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:focus { background-color: rgb(244,236,2); }</string>
   </property>
   <property name="text">
    <string>2、发送分站保存的压力传感器数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>291</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:focus { background-color: rgb(244,236,2); }</string>
   </property>
   <property name="text">
    <string>3、发送分站保存的传感器数据</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
